derive_builder/src/options/struct_mode.rs

```diff
@@ -40,6 +44,82 @@ impl StructMode {
         desc: "builder name",
         map: |x: String| { x },
     }
+    
+    impl_setter!{
+        ident: build_fn_name,
+        desc: "build function name",
+        map: |x: String| { x },
+    }
+    
+    #[allow(non_snake_case)]
+    fn parse_build_fn_options_metaItem(&mut self, meta_item: &syn::MetaItem) {
+        trace!("Build Method Options - Parsing MetaItem `{:?}`.", meta_item);
+        match *meta_item {
+            syn::MetaItem::Word(ref ident) => {
+                self.parse_build_fn_options_word(ident)
+            },
+            syn::MetaItem::NameValue(ref ident, ref lit) => {
+                self.parse_build_fn_options_nameValue(ident, lit)
+            },
+            syn::MetaItem::List(ref ident, ref nested_attrs) => {
+                self.parse_build_fn_options_list(ident, nested_attrs)
+            }
+        }
+    }
+    
+    #[allow(non_snake_case)]
+    fn parse_build_fn_options_nameValue(&mut self, ident: &syn::Ident, lit: &syn::Lit) {
+        trace!("Build fn Options - Parsing named value `{}` = `{:?}`", ident.as_ref(), lit);
+        match ident.as_ref() {
+            "name" => {
+                self.parse_build_fn_name(lit)
+            },
+            "skip" => {
+                self.parse_build_fn_skip(lit)
+            },
+            _ => {
+                panic!("Unknown build_fn option `{}` {}.", ident.as_ref(), self.where_diagnostics())
+            }
+        }
+    }
+    
+    /// e.g `private` in `#[builder(build_fn(private))]`
+    fn parse_build_fn_options_word(&mut self, ident: &syn::Ident) {
+        trace!("Setter Options - Parsing word `{}`", ident.as_ref());
+        match ident.as_ref() {
+            "skip" => {
+                self.build_fn_enabled = false;
```

Comment:
please use `self.build_fn_enabled(false)` instead. I know I've not done that consistently myself, though. ^^